.circleci/config.yml

```diff
@@ -175,6 +175,7 @@ references:
   remote_docker: &remote_docker
     setup_remote_docker:
       version: 20.10.6
+      docker_layer_caching: true
```

Comment:
Here is one change to `.circleci/config.yml` file that should be removed

Reply:
This is actually what is in the master branch. If I do not exactly match what is in this branch, the CI will fail...